cpp/cmake/thirdparty/get_cucollections.cmake

```diff
@@ -14,12 +14,14 @@
 
 # This function finds cuCollections and performs any additional configuration.
 function(find_and_configure_cucollections)
-  include(${rapids-cmake-dir}/cpm/cuco.cmake)
-  if(BUILD_SHARED_LIBS)
-    rapids_cpm_cuco(BUILD_EXPORT_SET cudf-exports)
-  else()
-    rapids_cpm_cuco(BUILD_EXPORT_SET cudf-exports INSTALL_EXPORT_SET cudf-exports)
-  endif()
+  rapids_cpm_find(
+    cuco 0.0.1
+    GLOBAL_TARGETS cuco::cuco
+    CPM_ARGS
+    GIT_REPOSITORY https://github.com/NVIDIA/cuCollections.git
+    GIT_TAG fabd885327dc1662e1a63d9b8870ebf888893c09
+    OPTIONS "BUILD_TESTS OFF" "BUILD_BENCHMARKS OFF" "BUILD_EXAMPLES OFF"
+  )
```

Comment:
Is this a temporary change?

Reply:
yes, will be reverted once the rapids-cmake PR is merged.